packages/ocaml-freestanding/ocaml-freestanding.0.4.3/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Martin Lucina <martin@lucina.net>"
+authors: "Martin Lucina <martin@lucina.net>"
+homepage: "https://github.com/mirage/ocaml-freestanding"
+bug-reports: "https://github.com/mirage/ocaml-freestanding/issues/"
+license: "MIT"
+tags: "org:mirage"
+dev-repo: "git+https://github.com/mirage/ocaml-freestanding.git"
+build: [make]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: [
+  "conf-pkg-config"
+  "ocamlfind" {build}
+  "ocaml-src" {build}
+  ("solo5-bindings-hvt" | "solo5-bindings-virtio" | "solo5-bindings-muen" | "solo5-bindings-genode")
+  "ocaml" {>= "4.04.2" & < "4.08.0"}
+]
+substs: [
+  "flags/cflags.tmp"
+  "flags/libs.tmp"
+]
+conflicts: [
+  "sexplib" {= "v0.9.0"}
+  "solo5-kernel-ukvm"
+  "solo5-kernel-virtio"
+  "solo5-kernel-muen"
```

Comment:
```suggestion
  "ocaml-system"
```

Reply:
I'd rather be in favour to close this PR and release ocaml-freestanding without the symlink (as PR'ed in https://github.com/mirage/ocaml-freestanding/pull/51, same for xen https://github.com/mirage/mirage-platform/pull/208)

Reply:
He someone is keen to do the work, yes sur no pb :p if not we can just merge this PR (as we did for mirage-platform) and wait for the next release to get support for the system compiler.